### ISSUE TITLE
Can't launch server

### ISSUE THREAD

**Issue by Beryyseimith** [Regalis11/Barotrauma#3666]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
I am unable to start a server from the main menu of the game, I've tried to uninstall, verify and reset my game files, all of which have failed.
Issue has been
**Steps To Reproduce**
-Host server
-Start

**Version**
Game version is 0.10.4.0 - OS is Windows Version 2004 (Newest windows update)
**Additional information**
N/A

**Comment by Regalis11** [comment]
Hi and thank you for the report!

When the server crashes, it should create a file called "servercrashreport.log" in the game's installation folder. If you can find that find and send it to us, it'd help us a lot in diagnosing the issue. Knowing the exact error message you're getting when you try to launch the server could also gives us some clues. Sorry for the inconvenience!

**Comment by Beryyseimith** [comment]
[servercrashreport.log](https://github.com/Regalis11/Barotrauma/files/5150109/servercrashreport.log)
Hi, thank you for the fast reply!
This was the latest report I could find, although it seems a bit outdated, if you need a later version, let me know.
Thank you again!

**Comment by mrjpjr** [comment]
Hi Beryyseimith, when you say "unable to start a server", is it taking a long time to connect, then giving you a "could not connect" error?
If so, I had the same issue, and you can try my workaround.
For me, the issue seemed to be caused by the memory requirements to run the game due to number of subs I had (all subs are loaded at startup).
Once I removed (backed up/moved) the sub files I was not using, the game loaded much faster, and I was able to host again.

**Comment by Regalis11** [comment]
Thank you for the report, but unfortunately it seems to be unrelated to this issue: it seems to be a crash during a running multiplayer round, caused by a bug that has since been fixed.

What kind of an error message do you see when you fail to start the server? I'm wondering if it's a crash at all or if the client just fails to connect to the server for some reason.

**Comment by mrjpjr** [comment]
This was the error, but no crash report was ever generated
![host server error](https://user-images.githubusercontent.com/58775950/92139093-42a6bb80-eddd-11ea-865c-c0f27e2b6955.jpg)

**Comment by WharkOddity** [comment]
I had the same problem, and still having it sometimes but less. I used to reset my whole laptop and reinstall steam and barotrauma. Also the problem come back again hardly ever, it come back after my laptop went in sleeping mode while barotrauma being open.

**Comment by Regalis11** [comment]
It could be possible that some kind of an antivirus software is interfering with the server.

Something that could be worth a try is launching the server directly from the game folder (a file called "DedicatedServer.exe"). This should open the server in a (text-only) command line interface, and it should show any errors that might be stopping the server from launching properly.

**Comment by R0merol** [comment]
Hi. I'm still having the same exact problem of not being able to host. I also can't find the servercrashreport.log. I don't have any mods installed. I tried reinstalling, moving the game folder, disabling the antivirus, to no avail. I tried running "DedicatedServer.exe" but it shows no error. Interestingly, I tried installing and running the game on another laptop and it was able to host the game no problem even with the same wi-fi. I'm only having this problem on this laptop. Any help would be greatly appreciated because I am desperately wanting to host on this laptop
